.prettierignore

```diff
@@ -0,0 +1,10 @@
+# Prettier will format everything, except what is explicitly specified here.
+
+# npm
+node_modules/
```

Comment:
prettier would ignore `node_modules` folder by default, so you don't need to list it in `.prettierignore`. See [File patterns](https://prettier.io/docs/en/cli.html#--ignore-path) section

Reply:
Ok, thank you.